.travis.yml

```diff
@@ -1,14 +1,16 @@
 sudo: false
 language: php
 php:
-    - 5.4
+    - 7.2
+    - 7.3
+    - 7.4
 
 branches:
     only:
         - master
 
 before_script:
-    - composer install --prefer-source
+    - composer update
```

Comment:
why composer update ?

Reply:
No special reason. It'll behave the same as `composer install`, given that `composer.lock` is git ignored.

I suspect that the `--prefer-source` used to mitigate a composer bug and is not relevant anymore.

Reply:
okay